common/src/main/scala/explore/utils/pprinter.scala

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.utils
+
+import cats.Monad
+import cats.syntax.all._
+import org.typelevel.log4cats.Logger
+import pprint.PPrinter
+
+import scala.scalajs.LinkingInfo
+
+trait pprinter {
+  def apply(x: Any): String
+}
+
+class devPPrinter extends pprinter {
+  private val pprinterInstance: PPrinter =
+    PPrinter(defaultHeight = 200, colorApplyPrefix = fansi.Color.Blue)
+
+  def apply(x: Any): String =
+    pprinterInstance(x, initialOffset = 4).toString
+}
+
+class prodPPrinter extends pprinter {
+  def apply(x: Any): String = x.toString
+}
+
+object pprinter {
+  private val printer: pprinter =
```

Comment:
Is `pprinter` not working in prod? or is it to minimize bundle size?

Reply:
Right, in order to minimize the bundle size in prod, pprinter defaults to toString in prod.